cgmanifest.json

```diff
@@ -49,7 +49,7 @@
          "component":{
             "type":"git",
             "git":{
-               "commitHash":"fdb978a580bcb461ea077b9772c973928b8dbf5f",
```

Comment:
Please revert this change

Reply:
Good catch, I didn't mean to check that in.